CMakeLists.txt

```diff
@@ -711,6 +739,20 @@ elseif(WIN32)
     install(FILES changelog.txt DESTINATION .)
     install(FILES README.md DESTINATION . RENAME ReadMe.txt)
     install(FILES LICENSE DESTINATION . RENAME License.TheXTech.txt)
+elseif(VITA)
+    include(cmake/package_vita.cmake)
+    if(ENABLE_FAST_MATH)
+        message("FAST MATH ON! HAVE FUN!")
+        set(CMAKE_C_FLAGS "${CMAKE_CXX_FLAGS} -ffast-math -mtune=cortex-a9 -mfpu=neon")
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ffast-math -mtune=cortex-a9 -mfpu=neon")
+    endif()
+    target_compile_definitions(thextech PRIVATE -DVITA -DVITA=1)
+    target_compile_definitions(thextech PRIVATE -DNO_SCREENSHOT)
```

Comment:
Some of these flags duplicate your previous VITA clause.

Reply:
Good catch!